trackpy/packing.py

```diff
@@ -0,0 +1,130 @@
+from scipy.spatial import cKDTree
+import random
+import numpy as np
+from warnings import warn
+
+
+
+def pairCorrelation2D(feat, fraction = 1., dr = .5, cutoff = 20., ndensity=None, boundary = None):
+    """
+    Calculate the pair correlation function in 2 dimensions.
+
+    Parameters
+    ----------
+    feat : Pandas DataFrame
+        DataFrame containing the x and y coordinates of particles
+    fration : float, optional
+        The fraction of particles to calculate g(r) with. May be used to increase speed of function. Particles selected at random.
+    dr : float, optional
+        The bin width
+    cutoff : float, optional
```

Comment:
I think this should not be an optional parameter. We don't know how closely spaced the particles are, so 20 could be completely inappropriate.